src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -140,6 +151,9 @@ public String toString(@Nullable Event e, boolean debug) {
 			case 3:
 				word = "struck";
 				break;
+			case 4:
+				word = "firework";
```

Comment:
```suggestion
				word = "launched";
```
see the return statement below